resources/tickets.py

```diff
@@ -87,7 +87,11 @@ class Tickets(Resource):
 
     @jwt_required
     def get(self):
-        return {'tickets': [ticket.json() for ticket in TicketModel.query.all()]}
+        data = Tickets.parser.parse_args()
+        if data["tenant"]:
+            return {'Tickets': [ticket.json() for ticket in TicketModel.find_by_tenantID(data["tenant"])]}
```

Comment:
Can we use the `find_by_id` method here?

Reply:
This is for a new feature where we display Tickets on the Tenant view, so we want to retrieve Tickets by Tenant-ID, not by Ticket-ID.

Reply:
ah I see. Thanks for clarifying!